data/filters/wg21.py

```diff
@@ -365,7 +365,7 @@ def rm():  _diff('rmcolor', 'sout', 'del')
 
     if 'sref' in elem.classes and isinstance(elem, pf.Span):
         target = pf.stringify(elem)
-        number = stable_names.get(target)
+        number = stable_names.get(target.split('#')[0])
```

Comment:
This looks great! Thanks for the fix. Can I suggest a minor change here where we do something like `name = target.split('#')[0]`, then the debug message to print "stable name <name> not found" rather than "stable name <target> not found"?